care/facility/api/viewsets/patient.py

```diff
@@ -220,7 +219,9 @@ def filter_by_bed_type(self, queryset, name, value):
     )
     ventilator_interface = CareChoiceFilter(
         field_name="last_consultation__last_daily_round__ventilator_interface",
-        choice_dict=VENTILATOR_CHOICES,
+        choice_dict={
```

Comment:
Please check this once and let me know if there is some better way to do this.

Reply:
This seems to be fine to me.

Reply:
@sainak